### PR TITLE
MF-130: Implement a form for adding and editing conditions

### PR DESCRIPTION
**Pull request by denniskigen**
Screenshot 2020-01-30 at 12 14 19

Screenshot 2020-02-03 at 23 17 59

### CODE REVIEW

**Comment by ojwanganto** [comment]
@denniskigen could we also plan for a similar widget that provides a grid for inputting multiple conditions? I think that way a user will be saved from the need to add conditions one by one. Just a light comment though

File: src/widgets/conditions/conditions-form.component.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+                    fontWeight: 500
+                  }}
+                >
+                  {patientCondition.resource.code.text}{" "}
+                </div>
+              </div>
+              <div className={`${styles.conditionHeader} omrs-bold`}>
+                <div
+                  className="omrs-type-body-regular"
+                  style={{ color: "#535355" }}
```
Comment (fatmali) [inline src/widgets/conditions/conditions-form.component.tsx]: use colors defined in the styleguide.

File: src/summaries/level-two-routes.component.tsx [hunk 2]
```diff
@@ -28,6 +29,21 @@ export const levelTwoRoutes: PatientChartRoute[] = [
     component: AllergyCardLevelThree,
     name: "Allergy"
   },
+  {
+    url: "/patient/:patientUuid/chart/conditions",
+    component: ConditionsBriefSummary,
```
Comment (fatmali) [inline src/summaries/level-two-routes.component.tsx]: this component was renamed to ConditionsOverview in master.

Comment (denniskigen) [reply src/summaries/level-two-routes.component.tsx]: ConditionsBriefSummary is the level-two component. ConditionsOverview is the master card.

File: src/widgets/conditions/conditions-form.component.tsx (hunk 1 above)
Comment (denniskigen) [reply src/widgets/conditions/conditions-form.component.tsx]: This particular color isn't included in the style guide I'm afraid.

**Comment by denniskigen** [comment]
> @denniskigen could we also plan for a similar widget that provides a grid for inputting multiple conditions? I think that way a user will be saved from the need to add conditions one by one. Just a light comment though

Sure @ojwanganto, I'm open to such a discussion although I see that this wasn't catered for in the design. I'll share this sentiment with Greg.

**Comment by denniskigen** [comment]
> This looks generally ok. However, I think that there is confusion over the naming conventions. The "level 1" card should be called conditions-summary.component.tsx and the component should be called ConditionsSummary. The level 2 component should be called conditions-detailed-summary.component.tsx and the component called ConditionsDetailedSummary.

#102 fixes this.

File: src/widgets/conditions/conditions-form.component.tsx (hunk 1 above)
Comment (denniskigen) [reply src/widgets/conditions/conditions-form.component.tsx]: I've removed this

**Comment by jonathandick** [comment]
Is this still a draft? Or is it ready to be reviewed for merging?

**Comment by denniskigen** [comment]
> Is this still a draft? Or is it ready to be reviewed for merging?

It's still a draft primarily because the conditions resource is not available either via OpenMRS or via the FHIR resource. Presently, all the conditions data is just mock data.

**Comment by jonathandick** [comment]
If that's the only reason, I think this can be converted to normal PR (not a draft) and we can merge

**Comment by fatmali** [comment]
@denniskigen you can mark this as ready for review so it can get merged.

**Comment by jonathandick** [comment]
I'm going to close this PR. Please push to new repo esm-patient-chart-widgets, once created

**Pull request closed without merging**